soundbay/conf/preprocessors/_preprocessors.yaml

```diff
@@ -4,6 +4,10 @@ _preprocessors:
     _target_: torchaudio.transforms.Spectrogram
     n_fft: ${data.n_fft}
     hop_length: ${data.hop_length}
+  min_freq:
```

Comment:
I think we should make it more informative.
`min_freq_filtering`?

Reply:
sounds right, changed the name